code/modules/mob/living/carbon/slime/_mood.dm

```diff
@@ -0,0 +1,5 @@
+#define POUT "pout"
+#define SAD "sad"
+#define ANGRY "angry"
+#define MISCHIEVOUS "mischevous"
```

Comment:
```suggestion
#define MISCHIEVOUS "mischievous"
```

Reply:
this is the original typo, i don't want to scramble around in dmi's and stuff to fix it right now